includes/admin/class-wcmypa-admin.php

```diff
@@ -971,6 +971,26 @@ public function showDeliveryDateForOrder(WC_Order $order): void
         $this->printDeliveryDate($deliveryOptions);
     }
 
+    /**
+     * @param $array
+     *
+     * @return array
+     */
+    private function flatten($array): array
+    {
+        $return = [];
+
+        foreach ($array as $key => $value) {
+            if (is_array($value)) {
+                $return = array_merge($return, $this->flatten($value));
+            }
+            else {
+                $return[$key] = $value;
+            }
+        }
+        return $return;
```

Comment:
Formatting: hoezo 'else' op een nieuwe regel? en return zonder witregel erboven. Het lijkt erop dat je later alleen op de values checkt (met in_array) dus de keys doen niet terzake. Kan je dan missch array_walk_recursive gebruiken dan ben je ook af van array_merge in een foreach (waar phpstorm over klaagt, dat het veel resources zou kosten).